manifest.json

```diff
@@ -14,11 +14,14 @@
             "css": [
                 "bootstrap-5.0.0-beta3-dist/css/bootstrap.min.css"
             ],
-            "js": ["bootstrap-5.0.0-beta3-dist/js/bootstrap.min.js"
+            "js": ["/content_scripts/storageLoader.js", "bootstrap-5.0.0-beta3-dist/js/bootstrap.min.js"
             ],
             "run_at": "document_end"
         }
     ],
+    "background": {
+      "scripts": ["popup/index.html"]  

```

Comment:
Why did you put under scripts, HTML file?

Reply:
My bad🤦🏻‍♂️🤦🏻‍♂️🤦🏻‍♂️, I'll delete it later. Good catch!